backend/dal.keycloak/Pims.Dal.Keycloak.csproj

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Mapster.DependencyInjection" Version="1.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Identity" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
```

Comment:
We usually don't "downgrade" packages - any particular reason for this?

Reply:
It was causing extra dependencies since the other Microsoft.Extensions.Logging libraries where on 5.0.0

Reply:
I see - good catch